server/planning/tests/events_service_test.py

```diff
@@ -22,6 +22,16 @@ def test_is_new_version():
 
     assert service.is_new_version(new_event, old_event)
 
+    new_event["subject"] = [{"qcode": "foo", "name": "Foo"}]
+    old_event["subject"] = [{"qcode": "foo", "name": "foo"}]
+
+    assert service.is_new_version(new_event, old_event)
+
+    new_event["subject"] = [{}]
+    old_event["subject"] = [{"qcode": "foo", "name": "foo"}]
+
+    assert service.is_new_version(new_event, old_event)
+
```

Comment:
Ideally, we should add a test scenario for subjects in different order.

Reply:
done